nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/HealthCheckExecutionOptions.java

```diff
@@ -23,11 +23,13 @@ and Distribution License("CDDL") (collectively, the "License").  You
     private boolean enabled;
     private long time;
     private TimeUnit unit;
+    private boolean displayOnHealthEndpoint;
```

Comment:
I am not sure about the wording. I think this is a flag to make the Payara Health Check appear on MP Health as well, isn't it?